crates/evm/execution-errors/src/lib.rs

```diff
@@ -134,6 +135,21 @@ pub enum BlockExecutionError {
     Internal(#[from] InternalBlockExecutionError),
 }
 
+/// Generic block execution error.
+pub trait GenericBlockExecutionError:
```

Comment:
```suggestion
pub trait BlockExecError:
```
let's rename this, remove the generic prefix and just change Execution -> Exec so rustc doesn't make a fuzz for type with same name